clouds/postgres/modules/quadbin/sql/__CARTO_ERROR.sql

```diff
@@ -0,0 +1,15 @@
+----------------------------
+-- Copyright (C) 2022 CARTO
+----------------------------
+
+CREATE OR REPLACE FUNCTION @@PG_SCHEMA@@.__CARTO_ERROR(
```

Comment:
Not needed now but these kind of functions can be moved to a utils module.

Reply:
Totally right, I would keep if for a future task